instrumentation/events/lttng-module/net.h

```diff
@@ -49,6 +148,22 @@ LTTNG_TRACEPOINT_EVENT_CLASS(net_dev_template,
 		ctf_integer_hex(void *, skbaddr, skb)
 		ctf_integer(unsigned int, len, skb->len)
 		ctf_string(name, skb->dev->name)
+		ctf_enum(net_transport_header, unsigned char, transport_header_type, __get_transport_header_type(skb))
+		ctf_custom_field(
```

Comment:
not sure why we don't see both the ipv4/ipv6 and transport headers all in the same event ?